src/transformers/models/recurrent_gemma/modeling_recurrent_gemma.py

```diff
@@ -658,7 +660,9 @@ def __init__(self, config: RecurrentGemmaConfig):
         self.final_norm = RecurrentGemmaRMSNorm(config.hidden_size, eps=config.rms_norm_eps)
         self.gradient_checkpointing = False
 
-        self.register_buffer("normalizer", torch.tensor(self.config.hidden_size**0.5, dtype=torch.bfloat16))
+        self.register_buffer(
+            "normalizer", torch.tensor(self.config.hidden_size**0.5, dtype=torch.bfloat16), persistent=False
+        )
```

Comment:
We don't need this to be persistant. This fixes an issue that we get with accelerate too.

Reply:
Good catch